pkgs/top-level/all-packages.nix

```diff
@@ -12955,9 +12955,9 @@ with pkgs;
   clangMultiStdenv = overrideCC stdenv buildPackages.clang_multi;
   multiStdenv = if stdenv.cc.isClang then clangMultiStdenv else gccMultiStdenv;
 
-  gcc_debug = lowPrio (wrapCC (gcc.cc.override {
-    stripped = false;
-  }));
+  gcc_debug = lowPrio (wrapCC (gcc.cc.overrideAttrs (oa: {
```

Comment:
```suggestion
  gcc_debug = lowPrio (wrapCC (gcc.cc.overrideAttrs (_: {
```

Reply:
Applied and squashed manually. (Failed to apply from github UI).